src/Lorentz/Contracts/BaseDAO/NonFA2.hs

```diff
@@ -0,0 +1,75 @@
+-- SPDX-FileCopyrightText: 2020 TQ Tezos
+-- SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+-- | Proposal related entrypoints
+module Lorentz.Contracts.BaseDAO.NonFA2
```

Comment:
`NonFA2` sounds too general because for example `Management` is non-FA2 as well. Here we have non-FA2 **token** functions. What if we call this module `Lorentz.Contracts.BaseDAO.Token` and move FA2 functionality into `Lorentz.Contracts.BaseDAO.Token.FA2`?